README.md

```diff
@@ -29,7 +29,8 @@ new mapboxgl.Compare(before, after, {
 
 ```js
 compare = new mapboxgl.Compare(before, after, {
-  mousemove: true // Optional. Set to true to enable swiping during cursor movement.
+  mousemove: true, // Optional. Set to true to enable swiping during cursor movement.
+  orientation: 'horizontal' // Optional. Sets the orientation of swiper, defaults to vertical
```

Comment:
Given that vertical is the default and maybe more popular, and that many people just copy and paste examples, I think it makes more sense to set this to vertical.